infra/events/src/main/java/com/simprints/infra/events/EventRepositoryImpl.kt

```diff
@@ -86,6 +86,12 @@ internal open class EventRepositoryImpl @Inject constructor(
         eventLocalDataSource.loadEventScope(downSyncEventScopeId)
 
     override suspend fun closeEventScope(eventScope: EventScope, reason: EventScopeEndCause?) {
+        if (eventScope.projectId == PROJECT_ID_FOR_NOT_SIGNED_IN) {
```

Comment:
We don't care about any events while not logged in?

Reply:
Only the events that are of a failed login sessions, see the Slack thread [here](https://simprints.slack.com/archives/C01BNE51M1T/p1711621715982659)

Reply:
As per the thread - for now, we should only preserve the events in sessions with a valid project id.